app/controllers/posts_controller.rb

```diff
@@ -17,26 +16,26 @@ def add_profile_username
     if @user != nil
       @profile = @user.profile
     end
-      if @profile == nil
-          respond_to do |format|
-         format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'}}
-         end
-      elsif @profile.name.blank?          
-          redirect_to council_posts_path(@council), flash: {alert: 'Användaren :"' + @user.username.to_s + '" måste fylla i fler uppgifter i sin profil.'} 
-      elsif @profile.posts.include?(@post)
-         redirect_to council_posts_path(@council), flash: {alert: @profile.name.to_s + '(' + @user.username.to_s + ') har redan posten '+@post.title.to_s + '.'}         
-      elsif (@post.limit != nil) && (@post.profiles.size >= @post.limit)
-         redirect_to council_posts_path(@council), flash: {alert: @post.title.to_s + ' har sitt maxantal.'}           
-      else 
-        @post.profiles << @profile        
-        redirect_to council_posts_path(@council), notice: @profile.name.to_s + ' (' + @profile.user.username.to_s + ') tilldelades posten '+@post.title.to_s + '.'
-        if (@profile.first_post == nil)   
-          @profile.update(first_post: @post.id)
-        end            
+    if @profile == nil
+      respond_to do |format|
+        format.html { redirect_to council_posts_path(@council), flash: {alert: 'Hittade ingen användare med det användarnamnet.'}}
```

Comment:
Line is too long. [130/100]<br>Space missing inside }.<br>Space inside { missing.<br>Space inside } missing.